R/method-fortify.R

```diff
@@ -394,11 +394,15 @@ fortify.phylo4 <- function(model, data, layout="rectangular", yscale="none",
 ##' @method fortify phylo4d
 ##' @export
 fortify.phylo4d <- function(model, data, layout="rectangular", yscale="none",
-                            ladderize=TRUE, right=FALSE, mrsd=NULL, ...) {
+                            ladderize=TRUE, right=FALSE, branch.length="branch.length",
```

Comment:
If you look careful for other methods, you will find they all missing the `branch.length` parameter.

This is not needed as it will be passed by `...`.